tests/integration/security/rbac/v2/basic/testdata/istio-rbac-grpc-rules.yaml.tmpl

```diff
@@ -0,0 +1,71 @@
+# istio-rbac-grpc-rules.yaml to enforce access control for gRPC services using Istio RBAC v2 rules.
+
+# For service a:
+# * Allow b to call a's Echo method.
+# * Do not allow c to talk to a (since NonExistingMethod does not exist).
+# * Allow d to call any methods of a.
+
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: access-a-grpc-echo
```

Comment:
Can we remove "a" from the name of the role? Since we did not mention service "a" in ServiceRole, the same role can be used for other services. Mentioning service name in the role can cause confusion. Same applies to other roles.

Reply:
In the test framework, you can start Echo service named a, b, c, etc. I think we conventionally write "For service ..., it allows/denies,..." (we define ServiceRole and AuthorizationPolicy for a service next to each other) to help test readers understand what's being tested. https://github.com/istio/istio/blob/master/tests/integration/security/rbac/v2/basic/testdata/istio-rbac-v2-rules.yaml.tmpl#L3-L29

Reply:
You can keep "a" if it helps user to understand the test case better. So "a" inside a role definition indicates where the role is going to be used. 

We just need to make sure that the ServiceRole named "a" should not be used for service "b" in AuthorizationPolicy, otherwise it will be very confusing. In the case of role reuse, that is, the same ServiceRole definition applied to different services (say, a, b, c), the name of the role need to contain all the services ("a", "b", "c").

Reply:
> We just need to make sure that the ServiceRole named "a" should not be used for service "b" in AuthorizationPolicy

I'm not sure I follow but in the AuthorizationPolicy test we use workload selector to select service "a" only.

Reply:
Currently it is. But if we add test cases for role reuse (apply the same role for different services, eg., a, b, c), we have to list all the services that the role may be used for (e.g., access-a-b-c-grpc-echo).

Reply:
Got it. Updated. Thanks!